--- pigz-2.4/Makefile.~1~	2017-12-26 16:35:01.000000000 +0000
+++ pigz-2.4/Makefile	2018-05-26 11:05:51.528283377 +0000
@@ -9,7 +9,7 @@ ZOP=deflate.o blocksplitter.o tree.o lz7
 
 pigz: pigz.o yarn.o try.o $(ZOP)
 	$(CC) $(LDFLAGS) -o pigz pigz.o yarn.o try.o $(ZOP) $(LIBS)
-	ln -f pigz unpigz
+	ln -sf pigz unpigz
 
 pigz.o: pigz.c yarn.h try.h $(ZOPFLI)deflate.h $(ZOPFLI)util.h
 
@@ -98,6 +98,14 @@ docs: pigz.pdf
 pigz.pdf: pigz.1
 	groff -mandoc -f H -T ps pigz.1 | ps2pdf - pigz.pdf
 
+install:
+	@mkdir -p ../prototype/{i386,amd64}/usr/bin ../prototype/{i386,amd64}/usr/share/man/man1
+	@install -m0755 pigz ../prototype/amd64/usr/bin
+	@install -m0755 unpigz ../prototype/amd64/usr/bin
+	@install -m0644 pigz.1 ../prototype/amd64/usr/share/man/man1
+	@install -m0755 pigz ../prototype/i386/usr/bin
+	@install -m0644 pigz.1 ../prototype/i386/usr/share/man/man1
+
 all: pigz pigzj pigzt pigzn docs
 
 clean:
