--- torque-6.1.0.orig/configure.ac	2017-01-20 20:33:26.193213877 +0100
+++ torque-6.1.0/configure.ac	2017-01-20 20:33:01.895826587 +0100
@@ -792,10 +792,8 @@
 dnl
 dnl we need libxml2
 dnl
-xmlLib=`xml2-config --libs | sed 's/-L@<:@^@<:@:space:@:>@@:>@* //g;s/-l//'`
 
-dnl skip the first two chars because its -l<libname>
-AC_CHECK_LIB(${xmlLib}, xmlGetProp,
+AC_CHECK_LIB(xml2, xmlGetProp,
   [],
   [AC_MSG_ERROR([TORQUE needs libxml2-devel in order to build]) ])
 
